src/index.html

```diff
@@ -9,6 +9,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a 
+        href="#home"
+        class="logo__link"
+      >
+        <img 
+          src="/images/logo.png" 
+          alt="moyo-logo"
+          class="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list-item active">
+            <label for="apple">
+              <a href="url" 
+                class= "link is-active"
+                id="apple"
+              >Apple</a>
+            </label>
+          </li>
+          <li class="list-item">
```

Comment:
```suggestion
      </a>
      
      <nav class="nav">
        <ul class="nav__list">
          <li class="list-item active">
            <label for="apple">
              <a href="url" 
                class= "link is-active"
                id="apple"
              >
                Apple
              </a>
            </label>
          </li>
          
          <li class="list-item">
```
pay attention to the code style, fix it all over the page

Reply:
thank you. fixed